test/e2e/components/vl-map-draw-point-action.js

```diff
@@ -8,6 +8,15 @@ class VlMapDrawPointAction extends VlMapDrawAction {
       await actions.move({origin: map, x: x, y: y}).click().perform();
     });
   }
+
+  async drawOnCoordinate({x = 10, y= 10} = {x: 10, y: 10}) {
+    await super.draw(async () => {
+      const map = await this.getMap();
+      const coordinates = await map.getPixelFromCoordinate([x, y]);
+      const actions = this.driver.actions();
+      await actions.move({origin: map, x: coordinates.x, y: coordinates.y}).click().perform();
+    });
+  }
```

Comment:
Ik heb liever geen nieuwe functie om coordinaten te tekenen. Als we #240 opnemen en bijgevolg maar één functie zullen overhouden, is dat een breaking change voor afnemers van de e2e testen. Niet dat ik schrik heb van terechte breaking changes, maar in deze lijkt het me eenvoudig te voorkomen.

Ik stel ik voor dat we (zelf te definiëren) #240 meteen opnemen voor een release van deze PR. Normaal gezien zullen de wijziging zich beperken tot het omzetten van pixels naar coordinaten.

Reply:
Dan ruil je toch gewoon de ene breaking change in voor de andere? Ben niet mee waarom we niet EN op pixels EN op coordinaten zouden willen kunnen tekenen.

Reply:
Ja, ik laat deze voorlopig dan staan in deze PR denk ik.
Ik zie wel voordelen in teken methodes met pixels EN coordinaten, maar ik kan me neerleggen indien het uiteindelijk enkel coordinaten zouden worden.

Reply:
Prima, ik zal het opnemen in #240.